CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ set(PROJECT_SOURCES
     DeckBuilder/Source/DeckBuilder.cpp
     DeckBuilder/Source/Deck.cpp
     DeckBuilder/Source/Card.cpp
+    fakedeck.cpp
```

Comment:
zmien na camel case - FakeDeck.cpp i możesz mergować

Reply:
Zmieniłem, ale jak na złość, mimo że wpisuję `git add FakeDeck.cpp`, to mi nie dodaje. Natomiast gdy sprawdzam `git status`, to wyświetla mi, że plik `fakedeck.cpp` został zmodyfikowany i nie dodany. Poza tym, gdy klikam tab, aby powłoka wpisała za mnie resztę nazwy pliku, to mi to uzupełnia jako `fakedeck.cpp`, co jest dziwne, ponieważ zmieniłem nazwę tego pliku. Nawet sprawdziłem poza edytorem i jest `FakeDeck.cpp`.
Jak to robię w innej powłoce to mi normalnie uzupełnia nazwę pliku, ale i tak `git status` pokazuje, że tą samą wiadomość. Dlatego musiałem użyć `git add fakedeck.cpp`.
@KGebski0036 @jakubkaczmarski  @jacekjarosz55 @jaroslawroszyk
Czy mógłby ktoś za mnie zmienić nazwę pliku na PascalCase'ową, bo ja nie potrafię? :)

Reply:
Eh, najprawdopodobniej to wina Windowsa. Mam od niedawna nowego i np.: dolny pasek samoczynnie zmienił swój styl z jasnego na ciemny, 2 razy częściowo zgasł na sekundę ekran bez żadnego ostrzeżenia, raz ikony zniknęły z dolnego paska na sekundę, a nawet w skrzynce mailowej mam 2 zakładki "Skrzynka odbiorcza", w których znajduje się to samo.

Reply:
na przyszłość możesz spróbować  git mv old_filename new_filename